### ISSUE TITLE
Review InnerSource videos on YouTube in respect to auto generated translations

### ISSUE THREAD

**Issue by arnom-ms** [InnerSourceCommons/InnerSourceLearningPath#295]
In order to help focus our efforts, we need to understand how much manual work do we need to do ah=nd how much can be done =automatically by machines. In this case YouTube has auto-generated subtitles and the goal of this issue is to figure out if that feature is good enough to provide subtitles in various languages so we do not have to provide them.

**Comment by arnom-ms** [comment]
To use the auto generated subtitles:
1) choose the video
2) select "cc" at the bottom of the video frame
3) select the cog next to CC and select "subtitles (CC)   English (auto generated)"
4) start the video

I had a look at the YT functionality and 1st was impressed that the introduction video (https://www.youtube.com/watch?time_continue=18&v=tP6K5oF2IxQ) had great subtitles in English and in German, This video has English subtitles and it seems Google/Youtube can translate from there quite well.

However when I tried another video, like 04 Contributor role  (https://www.youtube.com/watch?v=wv3uylJC4K0) which did not have English subtitles (only auto-generated ones), the subtitles in  German were just plain awful and did not help to convey what was said. The auto-generated English subtitles however (again) showed great quality. 
Others showed a similar pattern.

This from my perspective leaves us 2 options:
a) we provide English subtitles and then use YT to do the heavy lifting to other languages for us
or
b) we provide subtitles for a core set of languages other than English and use auto-generation for the English subtitles

I would think option a is preferable as it scales, however we need some broader review of this by other native language speakers (French, Spanish, Chinese, Japanese) to see if the same behavior can be observed in those languages.
means we can use it for English subtitles, however for other languages we need to decide we spend the additional effort and create the subtitles in the

**Comment by lenucksi** [comment]
Thanks for the research @arnom-ms!
I've got two questions here: 

> b) we provide subtitles for a core set of languages other than English and use auto-generation for the English subtitles

Do I read this correct as "manually generate de/es/fr/zh/ja subtitles and have Google translate them to English"?

Regarding the ability to auto-activate subtitles - Have you tried this insight @tsadler1988 came up with here: https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/249#issuecomment-670441634 ?

**Comment by arnom-ms** [comment]
I did not know, **thanks @lenucksi to make the connection to #249**.  As far as b) is concerned, I listed this as possible option to create subtitles and upload them for all the languages we deem important to be covered that way. I don't believe this would be a good use of our time as it seems having provided ONE language as subtitle is enough for the automated translation to then create reasonable localization for other languages. In this spirit I would suggest we provide English subtitles and then let the YT translator do the on the fly translation for subtitles in other languages. Why using English? Using English gives us the largest pool of potential contributors IMHO, AND it is a relative simple language which also more likely helps the YT translator to get the translation right.
